examples/complete/main.tf

```diff
@@ -115,19 +115,18 @@ resource "ibm_is_security_group" "sg1" {
   vpc  = ibm_is_vpc.example_vpc.id
 }
 
-resource "ibm_is_virtual_endpoint_gateway" "pgvpe" {
-  name = "${var.prefix}-vpe-to-pg"
-  target {
-    crn           = module.postgresql_db.crn
-    resource_type = "provider_cloud_service"
-  }
-  vpc = ibm_is_vpc.example_vpc.id
-  ips {
-    subnet = ibm_is_subnet.testacc_subnet.id
-    name   = "${var.prefix}-pg-access-reserved-ip"
-  }
-  resource_group  = module.resource_group.resource_group_id
-  security_groups = [ibm_is_security_group.sg1.id]
+module "vpe" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-vpe-module?ref=v2.4.0"
+  prefix = "${var.prefix}-vpe-to-pg"
```

Comment:
Is this good? Seems odd to set prefix to the full name.

This would appear to result in the name ${var.prefix}-vpe-to-pg-vpc-${var.prefix}-postgres.

Would it be clean to just pass var.prefix and to use vpc_name for better control of the final names?